torch/_dynamo/variables/tensor.py

```diff
@@ -88,6 +88,8 @@ def __init__(
         stride=None,
         is_contiguous=None,
         specialized_value=None,
+        _typed_storage=None,
```

Comment:
...what exactly is this? This seems very questionable, because it's not a primitive type like size/stride

Reply:
This is the tensor's _typed_storage `torch.storage.TypedStorage` - do you want me to write a first class citizen handing for this that does not go through this init?

One thing we do here is just store the original _typed_storage, and then defer to a lambda variable below to make a TypedStorageVariable - if you like, we can try a different approach, like fakifying it eagerly, making TypedStorageVariable eagerly, and turning ops like `_resize_` below into ops.